astropy/units/quantity.py

```diff
@@ -222,8 +222,9 @@ def __new__(cls, value, unit=None, dtype=None, copy=True, order=None,
                             subok=True, ndmin=ndmin)
 
         # Maybe list/tuple of Quantity? short-circuit array for speed
-        if(not isinstance(value, np.ndarray) and isiterable(value) and
-           all(isinstance(v, Quantity) for v in value)):
+        if (not isinstance(value, np.ndarray) and isiterable(value) and
+            all(isinstance(v, Quantity) for v in value)):
```

Comment:
Yes, that is another way suggested in PEP8. But I'll leave that for another time!